lib/bindings/src/breez_sdk_liquid.udl

```diff
@@ -313,35 +324,59 @@ dictionary GetInfoResponse {
 };
 
 dictionary PrepareSendRequest {
-    string invoice;
+    string payment_destination;
+    u64? amount_sat;
+};
+
+[Enum]
+interface SendDestination {
+    BIP21(LiquidAddressData address_data);
+    Bolt11(LNInvoice invoice);
 };
 
 dictionary PrepareSendResponse {
-    string invoice;
+    SendDestination payment_destination;
     u64 fees_sat;
 };
 
+dictionary SendPaymentRequest {
+    PrepareSendResponse prepare_response;
+};
+
 dictionary SendPaymentResponse {
     Payment payment;
 };
 
+enum ReceiveMethod {
+    "BIP21",
+    "Bolt11",
+};
+
 dictionary PrepareReceivePaymentRequest {
-    u64 payer_amount_sat;
+    u64? payer_amount_sat;
+    bool use_lightning;
 };
 
 dictionary PrepareReceivePaymentResponse {
-    u64 payer_amount_sat;
+    u64? payer_amount_sat;
+    bool use_lightning;
     u64 fees_sat;
 };
 
 dictionary ReceivePaymentRequest {
-    PrepareReceivePaymentResponse prepare_res;
+    PrepareReceivePaymentResponse prepare_response;
     string? description = null;
 };
 
+[Enum]
+interface ReceiveDestination {
+    BIP21(string uri);
+    Liquid(string address):
+    Bolt11(string id, string invoice);
```

Comment:
Why do we need the id?

Reply:
In the previous receive response we exposed the swap id, but thinking about it now the user doesn't need to be aware of it

Reply:
Yes I also think we don't need it. Let's remove it and we are left only with the result string.